test/util/util.go

```diff
@@ -204,6 +205,18 @@ func DeleteRuntimeClass(ctx context.Context, c client.Client, runtimeClass *node
 	return nil
 }
 
+func UnholdQueue(ctx context.Context, k8sClient client.Client, cq *kueue.ClusterQueue) {
+	gomega.EventuallyWithOffset(1, func() error {
+		var cqCopy kueue.ClusterQueue
+		gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(cq), &cqCopy)).To(gomega.Succeed())
+		if ptr.Deref(cqCopy.Spec.StopPolicy, kueue.None) == kueue.None {
+			return nil
+		}
+		cqCopy.Spec.StopPolicy = ptr.To(kueue.None)
+		return k8sClient.Update(ctx, &cqCopy)
```

Comment:
```suggestion
	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
		var cqCopy kueue.ClusterQueue
		g.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(cq), &cqCopy)).To(gomega.Succeed())
		if ptr.Deref(cqCopy.Spec.StopPolicy, kueue.None) == kueue.None {
			return
		}
		cqCopy.Spec.StopPolicy = ptr.To(kueue.None)
		g.Expect(k8sClient.Update(ctx, &cqCopy)).To(gomega.Succeed())
```

I think it would be better to propagate the gomega instance.

Reply:
Done